twisted/test/test_nooldstyle.py

```diff
@@ -48,6 +48,11 @@ class SomeNewStyleClass(object):
 
 
 
+class SomeClassThatUsesOldStyle(SomeOldStyleClass):
```

Comment:
Since this class is only used inside `test_makesNewStyle`, perhaps it should just be defined in there, so it doesn't need (A) as much whitespace and (B) a docstring?